stdlib/universe/window_test.go

```diff
@@ -101,6 +101,20 @@ func TestFixedWindow_PassThrough(t *testing.T) {
 	})
 }
 
+func TestFixedWindow2_PassThrough(t *testing.T) {
```

Comment:
Can you add a Flux acceptance test too? The ones that are written in _test.flux files.

Reply:
Yup, good catch.